go/consensus/tendermint/registry/registry.go

```diff
@@ -125,10 +125,6 @@ func (tb *tendermintBackend) WatchRuntimes(ctx context.Context) (<-chan *api.Run
 	return typedCh, sub, nil
 }
 
-func (tb *tendermintBackend) GetNodeList(ctx context.Context, height int64) (*api.NodeList, error) {
-	return tb.getNodeList(ctx, height)
```

Comment:
the `WatchNodeLists` which now that i looked into it also isn't used anywhere, maybe we should just get rid of it all together yes, in any case i think it would be better extracting this in a separate  PR, so will remove the changes from here.